specification/attributes/string_format.md

```diff
@@ -0,0 +1,32 @@
+# String Format
+
+Columns that capture String values conforming to specified requirements foster data integrity, interoperability, and consistency, improve data analysis and reporting, and support more reliable data-driven decision-making.
+
+All columns capturing a String value, defined in the [FOCUS](#glossary:finops-cost-and-usage-specification) specification, MUST follow the requirements listed below. Custom String value capturing columns SHOULD adopt the same requirements over time.
+
+## Attribute ID
+
+StringFormat
+
+## Attribute Name
+
+String Format
+
+## Description
+
+Requirements for String-capturing columns appearing in billing data.
+
+## Requirements
+
+* String values appearing in the billing data MUST maintain the original casing, spacing, and other pertinent consistency factors.
+* String values referring to the same entity SHOULD be consistent within a provider's context.
+* The provider SHOULD ensure that both provider-defined and user-input String values do not contain multiple consecutive spaces or leading/trailing spaces.
```

Comment:
* The provider SHOULD ensure that both provider-defined and user-input String values do not contain multiple consecutive spaces or leading/trailing spaces.

Can we move this to supporting content as it can be confusing... we don't want providers to modify the data in any way from their internal datastore to the focus export and i am concerned this could be misinterpreted.